### ISSUE TITLE
Insufficient color contrast for elements

### ISSUE THREAD

**Issue by keenanamigos** [keenanamigos/random-quote-machine#5]
The quote, author, new quote button, and tweet button have insufficient contrast between the foreground and the background which is an accessibility issue.

**Comment by keenanamigos** [comment]
Fixed with commit #4378c22f4704c2e0fd1d8d1cc50b24cb17da05cb
